src/frontend/src/pages/CreateWorkPackagePage/CreateWorkPackageFormView.tsx

```diff
@@ -119,25 +126,30 @@ const CreateWorkPackageFormView: React.FC<CreateWorkPackageFormViewProps> = ({
 
   const blockedByFormControl = (
     <FormControl fullWidth>
-      <FormLabel>Blocked By</FormLabel>
-      {blockedBy.map((_element, i) => {
-        return (
-          <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
-            <TextField required autoComplete="off" {...register(`blockedBy.${i}.wbsNum`)} sx={{ width: 9 / 10 }} />
-            <IconButton type="button" onClick={() => removeBlocker(i)} sx={{ mx: 1, my: 0 }}>
-              <DeleteIcon />
-            </IconButton>
-          </Grid>
-        );
-      })}
-      <Button
-        variant="contained"
-        color="success"
-        onClick={() => appendBlocker({ wbsNum: '' })}
-        sx={{ my: 2, width: 'max-content' }}
-      >
-        + ADD NEW BLOCKER
-      </Button>
+      <Controller
+        name="blockedBy"
+        control={control}
+        render={({ field: { onChange, value: formValue } }) => (
+          <Autocomplete
+            isOptionEqualToValue={(option, value) => {
+              console.log(option, value);
```

Comment:
delete (and then the function can just be `(option, value) => option.id === value.id`